### PR TITLE
feature 7 computer player

### PR DESCRIPTION
**Pull request by kimseeum**
Implemented most of the algorithm to make the decision for a computer player, just need to make a few more methods. Also, I barely have any comments, and no tests at all, will do them later.

### CODE REVIEW

File: src/main/ComputerDecision.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Iterator;
+
+public class ComputerDecision {
+    private Card[] hand;
+    private Card[] flop;
+    private int callAmount;
+
+    public ComputerPlayerDecisionClass(Card[] hand, Card[] flop, int callAmount) {
```
Comment (enfz) [inline src/main/ComputerDecision.java:11]: Would probably want to move this to an entity interface and factory system if the decision is created on each new round.

**Review by enfz — APPROVED** [review]
The implementation looks good and no repetitive algorithm is involved. Since we are creating multiple instances of it, should consider separating parts a bit into an entity interface, factory, etc.

File: src/main/ComputerDecision.java (hunk 1 above)
Comment (kimseeum) [reply src/main/ComputerDecision.java:11]: I think making the class static could work, we don't really need different instances using the same algorithm, so I just need to make a static method that takes parameters for the cards.

**Review by Andy-Guo-21 — APPROVED** [review]
Logic looks fine. Need to add docstring, tests, and finish the todo, but otherwise looks good to merge.

**Review by Jefferson-liu — APPROVED** [review]
The finished code looks good right now, but docstrings will need to be added as Andy said, and also we still need to decide how this will interact with the player class to create an computer player.

**Pull request merged**